src/screens/Profile/Header/Shell.tsx

```diff
@@ -64,6 +68,10 @@ let ProfileHeaderShell = ({
 
   const aviRef = useAnimatedRef()
 
+  const atUri = useMemo(() => {
+    return AtUri.make(profile.did).href
```

Comment:
This could arguably be better as: 
```ts
return new AtUri(`at://${profile.did}/app.bsky.actor.profile/self`).href
```

Which would match the alternate link: https://github.com/bluesky-social/social-app/blob/028ed2f8e2edd1d35e62246e7e6df2cea09fdb07/bskyweb/templates/profile.html#L27